libRnfpp.doxygen.cfg

```diff
@@ -460,7 +460,7 @@ SUBGROUPING            = YES
 INLINE_GROUPED_CLASSES = NO
 
 # When the INLINE_SIMPLE_STRUCTS tag is set to YES, structs, classes, and unions
-# with only public data fields or simple typedef fields will be shown inline in
```

Comment:
This file was edited by accident, I suppose.  I will revert all changes to this file.